<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FiffRawViewSettingsWidget</class>
 <widget class="QWidget" name="FiffRawViewSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>0.950000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="label_timeSpacers">
       <property name="text">
        <string>Time spacer every (ms):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="m_comboBox_distaceTimeSpacer">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>500</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1000</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_signalColor">
       <property name="text">
        <string>Change signal color:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="m_pushButton_signalColor">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="m_spinBox_windowSize">
       <property name="suffix">
        <string> s</string>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="m_pushButton_backgroundColor">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_backgroundColor">
       <property name="text">
        <string>Change background color:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_numberChannels">
       <property name="text">
        <string>Number of channels:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_windowSize">
       <property name="text">
        <string>Window size:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QComboBox" name="m_comboBox_imageType">
       <item>
        <property name="text">
         <string>PNG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SVG</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="m_pushButton_makeScreenshot">
       <property name="text">
        <string>Make screenshot</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="m_doubleSpinBox_numberVisibleChannels">
       <property name="suffix">
        <string/>
       </property>
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>30.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
